DemoApp/DemoApp/ViewModels/TableViewModel.cs

```diff
@@ -0,0 +1,62 @@
+using CommunityToolkit.Mvvm.ComponentModel;
+using DemoApp.Constant;
+using DemoApp.Model;
+using Microsoft.Maui.Controls;
+
+namespace DemoApp.ViewModels;
+public partial class TableViewModel : ObservableObject
+{
+    [ObservableProperty]
+    private List<User> _users;
+    [ObservableProperty]
+    private User _selectedItem;
+    [ObservableProperty]
+    private bool _dividerToggled = false;
+    [ObservableProperty]
+    private SelectionMode _selectionMode = SelectionMode.None;
+    [ObservableProperty]
+    [NotifyPropertyChangedFor(nameof(SelectionMode))]
+    private int _selectionModeRadioOption = 0;
+    partial void OnSelectionModeRadioOptionChanging(int value)
+    {
+        SelectionMode = (SelectionMode)value;
+    }
+
+    public TableViewModel()
+    {
+        var user1 = new User()
```

Comment:
Can you move the data set to a json file and read it from there? In that way, if we want to add more user data, we can just edit the json file. Refer 'UserData.json' in the Raw folder.

Reply:
And refer the data from the UserData.json and use the usernames and logo images from there

Reply:
Done